codegen/writers/endpoint.go

```diff
@@ -29,19 +32,85 @@ type (
 	}
 )
 
+type endpointWriter struct {
+	sections   []*codegen.Section
+	outputPath string
+}
+
+func (e endpointWriter) Sections() []*codegen.Section {
+	return e.sections
+}
+
+func (e endpointWriter) OutputPath() string {
+	return e.outputPath
+}
+
 // EndpointsWriter returns the codegen.FileWriter for the endpoints of the given
 // service.
 func EndpointsWriter(api *design.APIExpr, service *design.ServiceExpr) codegen.FileWriter {
-	return nil
+	return endpointWriter{
+		sections: []*codegen.Section{
+			codegen.Header("", "endpoints", []*codegen.ImportSpec{
+				&codegen.ImportSpec{Path: "context"},
+				&codegen.ImportSpec{Path: "goa.design/goa.v2"},
+				&codegen.ImportSpec{Path: "goa.design/goa.v2/examples/account/gen/services"},
+			}),
+			Endpoint(api, service),
+		},
+		outputPath: "gen/endpoints/", // TODO Set output path.
```

Comment:
The output path is the relative path from the output path specified by user. So compute a filename from the service name (using Go conventions: lowercase and underscores to separate words) and append what you have above with it.

The services Go import path should be computed and I'm realizing now that we'll need the absolute output path for that (and use `build.Import`). So we should add the output path specified on the command line as an argument to `EndpointsWriter`.

The output path returned by the writer should still be relative though (not all writers may need the absolute path as input).